src/figuresClasses.ts

```diff
@@ -1,11 +1,85 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Fail data!!!');
```

Comment:
[CODE STYLE] - When throwing an error, provide a detailed description in the error message. Instead of 'Fail data!!!', a more descriptive message would be helpful, for example, 'Triangle sides must be positive numbers.'